### PR TITLE
Resolve auto merge conflicts for Branch 21.08 from branch 21.06

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR resolves conflicts for auto-merger to proceed: https://github.com/rapidsai/cudf/pull/8317

### CODE REVIEW

**Comment by kkraus14** [comment]
@galipremsagar when we fix the auto-merge we typically want to keep the history from the source branch so the history of `21.08` is compatible with `21.06`.

**Review by revans2 — APPROVED** [review]
The java changes look fine.

**Comment by galipremsagar** [comment]
rerun tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,325 @@ Please see https://github.com/rapidsai/cudf/releases/tag/v0.20.0a for the latest
 # cuDF 0.19.0 (Date TBD)
 
 Please see https://github.com/rapidsai/cudf/releases/tag/v0.19.0a for the latest changes to this development branch.
+# cuDF 21.06.00 (Date TBD)
+
+Please see https://github.com/rapidsai/cudf/releases/tag/v21.06.00a for the latest changes to this development branch.
+
+# cuDF 0.19.0 (21 Apr 2021)
```
Comment (ajschmidt8) [inline CHANGELOG.md:16]: can you check this file again? it doesn't look like the conflicts were resolved correctly. there should only be one `0.19.0` entry and the `0.20` entry should be replaced with `21.06.00`.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

**Review by ajschmidt8 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (galipremsagar) [reply CHANGELOG.md]: Thanks @ajschmidt8 !

**Review by ttnghia — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -67,13 +67,13 @@ cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html),
 
 For `cudf version == 21.081.06` :
 ```bash
-# for CUDA 10.1
+# for CUDA 11.0
 conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=21.081.06 python=3.7 cudatoolkit=10.1
+    cudf=21.081.06 python=3.7 cudatoolkit=11.0
```
Comment (kkraus14) [inline README.md:72]: the versions here look wrong

File: README.md [hunk 3]
```diff
@@ -67,13 +67,13 @@ cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html),
...
 
-# or, for CUDA 10.2
+# or, for CUDA 11.2
 conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=21.081.06 python=3.7 cudatoolkit=10.2
+    cudf=21.081.06 python=3.7 cudatoolkit=11.2
```
Comment (kkraus14) [inline README.md:76]: here as well

File: README.md (hunk 2 above)
Comment (galipremsagar) [reply README.md:72]: Resolving it in a separate PR: https://github.com/rapidsai/cudf/pull/8331

**Comment by kkraus14** [comment]
@galipremsagar you need to pull in the latest merged PRs from 21.06 now as they don't show in this PR.

**Comment by galipremsagar** [comment]
> @galipremsagar you need to pull in the latest merged PRs from 21.06 now as they don't show in this PR.

Pushed the latest changes

**Review by kkraus14 — APPROVED** [review]

**Pull request merged**